.remarkrc

```diff
@@ -1,6 +1,9 @@
 {
   "plugins": [
     "preset-lint-markdown-style-guide",
-    ["lint-no-duplicate-headings", false]
+    ["lint-no-duplicate-headings", false],
+    ["lint-list-item-indent", "tab-size"],
+    ["lint-emphasis-marker", "consistent"],
+    ["lint-maximum-line-length", false]
```

Comment:
Why are those changed to the linter necessary for the changelog generation?

Reply:
Those changes are necessary for the auto generated changelog (after it was corrected through remark) to pass the linter style guide